index.html

```diff
@@ -270,76 +287,100 @@ <h2>
+          When launching a web app via any means the trigger of the launch
+          should run the steps to [=launch a web app=] with a {{LaunchParams}}
+          containing relevant launch information (e.g. share data in the event
+          of a share target launch).
+        </p>
+        <p class="note">
+          Examples of different web app launch triggers:
+          <ul class="note">
+            <li>OS application launch surfaces
+            </li>
+            <li><a href="https://wicg.github.io/manifest-incubations/index.html#protocol_handlers-member">Protocol handling</a>
+            </li>
+            <li><a href="https://wicg.github.io/manifest-incubations/index.html#file_handlers-member">File handling</a>
+            </li>
+            <li><a href="https://wicg.github.io/manifest-incubations/index.html#share_target-member">Share target</a>
+            </li>
+            <li>Link capturing
+            </li>
+          </ul>
         </p>
         <p>
-          The steps to <dfn>launch a web app</dfn> are given by the following algorithm and takes a {{LaunchParams}} |params:LaunchParams|.
+          The steps to <dfn>launch a web app</dfn> are given by the following
+          algorithm and takes an optional {{LaunchParams}}
+          |params:LaunchParams|.
         </p>
         <ol class="algorithm">
-          <li>If |params| is null, set |params| to a new {{LaunchParams}} with {{LaunchParams/targetURL}} set to [=manifest/start_url=].
+          <li>If |params| is null, set |params| to a new {{LaunchParams}} with
+              {{LaunchParams/targetURL}} set to [=manifest/start_url=].
           </li>
-          <li>Set |client| to the result of running the steps to [=prepare a web app launch client=] passing [=manifest/route_to=] and |params|.{{LaunchParams/targetURL}}.
+          <li>Set |client| to the result of running the steps to
+              [=prepare a web app launch client=] passing [=manifest/route_to=]
+              and |params|.{{LaunchParams/targetURL}}.
           </li>
-          <li>Append |params| to the [=unconsumed launch params=] of the |client|'s document's {{Window.LaunchQueue}}.
+          <li>Append |params| to the [=unconsumed launch params=] of the
+              |client|'s document's {{Window/launchQueue}}.
           </li>
-          <li>If the [=assigned launch consumer=] |consumer| is set:
-            <ol>
-              <li>[=list/For each=] |launch_params:LaunchParams| of [=unconsumed launch params=]:
-                <ol>
-                  <li>Invoke |consumer| with |launch_params|.
-                  </li>
-                </ol>
-              </li>
-              <li>Set [=unconsumed launch params=] to the empty list.
-              </li>
-            </ol>
+          <li>Run the steps to [=process unconsumed launch params=].
```

Comment:
add "... of the client doucment's launchQueue" (see below).

Reply:
Added.